3rdparty/communi/src/util/util.pri

```diff
@@ -0,0 +1,43 @@
+######################################################################
+# Communi
+######################################################################
+
+DEFINES += BUILD_IRC_UTIL
+
+INCDIR = $$PWD/../../include/IrcUtil
+
+DEPENDPATH += $$PWD $$INCDIR
+INCLUDEPATH += $$PWD $$INCDIR
+
+CONV_HEADERS += $$INCDIR/IrcCommandParser
+CONV_HEADERS += $$INCDIR/IrcCommandQueue
+CONV_HEADERS += $$INCDIR/IrcCompleter
+CONV_HEADERS += $$INCDIR/IrcLagTimer
+CONV_HEADERS += $$INCDIR/IrcPalette
+CONV_HEADERS += $$INCDIR/IrcTextFormat
+CONV_HEADERS += $$INCDIR/IrcUtil
+
+PUB_HEADERS += $$INCDIR/irccommandparser.h
+PUB_HEADERS += $$INCDIR/irccommandqueue.h
+PUB_HEADERS += $$INCDIR/irccompleter.h
+PUB_HEADERS += $$INCDIR/irclagtimer.h
+PUB_HEADERS += $$INCDIR/ircpalette.h
+PUB_HEADERS += $$INCDIR/irctextformat.h
+PUB_HEADERS += $$INCDIR/ircutil.h
+
+PRIV_HEADERS  = $$INCDIR/irccommandparser_p.h
+PRIV_HEADERS  = $$INCDIR/irccommandqueue_p.h
```

Comment:
These files like the directory implies are part of the upstream communi library - any changes to them should be filed with upstream as well, otherwise we'd have to maintain out local patches to them.